src/v/kafka/server/group.h

```diff
@@ -112,7 +112,7 @@ kafka::error_code map_store_offset_error_code(std::error_code);
 /// \brief A Kafka group.
 ///
 /// Container of members.
-class group {
+class group : public ss::enable_lw_shared_from_this<group> {
```

Comment:
Final?

Reply:
@senior7515 - sure. I saw this in a few places with shared_from_this.

Is it because any subclass would break b/c of the CRTP pattern where `ss::enable_lw_shared_from_this<group>` knows the type and so would create the wrong pointer type?

Reply:
Ah I see.